.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     working_directory: ~/normandy
 
     environment:
-        COMPOSE_FILE: ci/docker-compose.yml
+      COMPOSE_FILE: ci/docker-compose.yml
```

Comment:
My vscode fixes yaml files automatically and although this is unrelated I don't think it hurts to correct.

Reply:
In the future it would be good to only include these kind of fixes to files you fixed already. This change is more correct so lets keep it, but if we're going to have automatic fixes for these kind of files, we should add them as rules to Therapist instead of ad-hoc changes.